### PR TITLE
Connect Mixpanel

### PR DESCRIPTION
**Pull request by dimarconicola**
##### Description
I connected Mixpanel and tracked main CTAs and links currently present on the website.

##### Checklist

- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Testing
It won't break anything. I checked that all events appeared on the live view section of Mixpanel

##### Refers/Fixes

Refs: #19

### CODE REVIEW

File: src/Components/BigBlackTitle.js [hunk 1]
```diff
@@ -1,82 +1,98 @@
...
 
 const Container = styled.div`
-display: flex;
-flex-direction: column;
-justify-content: center;
-align-items: center;
-font-family: var(--secondary-font);
-background-color: var(--white);
-padding: 10% 0;
+  display: flex;
```
Comment (Alexintosh) [inline src/Components/BigBlackTitle.js:6]: Indentation issue

File: src/Components/BigBlackTitle.js [hunk 2]
```diff
@@ -1,82 +1,98 @@
...
-  
-  export default BigBlackTitle;
\ No newline at end of file
+  return (
+    <Container>
+      <section className="content center">
+        <Title>We promise it's good stuff. Read the Whitepaper</Title>
+        <A
+          onClick={() =>
+            link({ position: "navbar", to: "whitepaper", type: "text" })
```
Comment (Alexintosh) [inline src/Components/BigBlackTitle.js:81]: to: whitepaper-request would be better as we are not linking the actual whitepaper just yet

File: src/Components/BigBlackTitle.js [hunk 3]
```diff
@@ -1,82 +1,98 @@
...
+          }
+          className="navbar-item"
+          href="https://github.com/Alexintosh/whitepaper"
```
Comment (Alexintosh) [inline src/Components/BigBlackTitle.js]: this should NOT be there, it needs to link to the newsletter form or maybe a typeform?

File: src/Components/FAQ.js [hunk 4]
```diff
@@ -118,7 +143,15 @@ const FAQ = props => {
           </SingleItem>
         </Gradient>
         <Gradient>
-          <SingleItem>
+          <SingleItem
+            onClick={() => {
+              cta({
```
Comment (Alexintosh) [inline src/Components/FAQ.js]: I'm not sure this is a CTA, looks more like a click event

File: src/Components/PrimaryButton.js [hunk 5]
```diff
@@ -42,12 +40,21 @@ background-size: 100%;
...
-  };
-  
-  export default PrimaryButton;
\ No newline at end of file
+  return (
+    <Button
+      onClick={() => {
+        cta({
+          position: "hero",
+          to: "/Investment",
```
Comment (Alexintosh) [inline src/Components/PrimaryButton.js]: let's please use lowercase links

File: src/Components/TopNavi.js [hunk 6]
```diff
@@ -72,24 +73,66 @@ const TopNavi = props => {
...
         </Link>
       </Left>
       <Right>
-        <A href="#">whitepaper</A>
+        <A
+          onClick={() =>
+            link({ position: "navbar", to: "whitepaper", type: "text" })
+          }
+          className="navbar-item"
+          href="https://github.com/Alexintosh/whitepaper"
```
Comment (Alexintosh) [inline src/Components/TopNavi.js]: Please DON'T link the whitepaper

File: src/Components/TopNavi.js [hunk 7]
```diff
@@ -72,24 +73,66 @@ const TopNavi = props => {
...
+              link({ position: "navbar", type: "text", to: "charts" })
+            }
+          >
+            charts
+          </StyledLink>
         </MobileNone>
         <MobileNone>
-          <StyledLink to="/Investment">investment</StyledLink>
+          <StyledLink
+            to="/Investment"
```
Comment (Alexintosh) [inline src/Components/TopNavi.js]: Uppercase link

File: src/Components/TopNavi.js [hunk 8]
```diff
@@ -72,24 +73,66 @@ const TopNavi = props => {
...
+          </StyledLink>
         </MobileNone>
         <MobileNone>
-          <A href="#">discord</A>
+          <A
+            onClick={() =>
+              link({ position: "navbar", to: "discord", type: "text" })
+            }
+            className="navbar-item"
+            href="https://discord.gg/CdzWXc"
```
Comment (Alexintosh) [inline src/Components/TopNavi.js]: Maybe we should have a similar redirect like `pie.network/discord`

File: src/Components/TopNavi.js [hunk 9]
```diff
@@ -72,24 +73,66 @@ const TopNavi = props => {
...
         <MobileNone>
           {" "}
-          <A href="#">contact</A>
+          <A
+            onClick={() =>
+              link({ position: "navbar", to: "contact", type: "text" })
+            }
+            className="navbar-item"
+            href="https://discord.gg/CdzWXc"
+          >
```
Comment (Alexintosh) [inline src/Components/TopNavi.js:133]: Same to upstairs

File: src/Components/TopNavi.js [hunk 10]
```diff
@@ -72,24 +73,66 @@ const TopNavi = props => {
...
+            contact
+          </A>
```
Comment (Alexintosh) [inline src/Components/TopNavi.js]: A should be lowecase

**Review by Alexintosh — DISMISSED** [review]
Please check my comment and ask @dmvt or me for help.

**Review by MickdeGraaf — COMMENTED** [review]
Other than the comments @Alexintosh made it looks good to me.

File: src/Components/BigBlackTitle.js (hunk 3 above)
Comment (dimarconicola) [reply src/Components/BigBlackTitle.js]: I'm linking it to a typeform

File: src/Components/TopNavi.js (hunk 10 above)
Comment (dimarconicola) [reply src/Components/TopNavi.js]: it's a styled component

**Review by dmvt — APPROVED** [review]

**Pull request merged**